vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -27,6 +27,8 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import org.slf4j.LoggerFactory;
```

Comment:
Nit: Order of this import could be aligned with how the other imports are structured.

Reply:
Seems like this particular import is all over the place. In some cases it's before vaadin component imports, in others it's after. If it's important, should we do a PR that specifically updates the order of it?

Reply:
Not really important, let's leave as is. The imports in the codebase are basically ordered by whatever the respective developers IDE thinks is best. 

If we want to do something we should maybe have an automated way of ordering them, similar to the code formatter. Like in `web-components` for example where a linter that checks order of imports and can fix them if needed.